packages/xo-server-sdn-controller/src/index.js

```diff
@@ -222,6 +254,33 @@ class SDNController extends EventEmitter {
             if (center !== undefined) {
               this._starCenters.set(center.$id, center.$ref)
             }
+
+            const crossPoolNetworkUuid =
+              network.other_config.cross_pool_network_uuid
```

Comment:
You should add some doc explaining which properties you are using, eg: https://github.com/vatesfr/xen-orchestra/blob/77c62d6e7d8de35f4d0b71b39678b4a17d5d4a63/packages/xo-server/src/xo-mixins/backups-ng/index.js#L478-L498

Reply:
Can I do it in this PR or should I do a separate one after the merge? I ask because some fields are not relevant to this PR.

Reply:
As you wish.